README.md

```diff
@@ -574,6 +628,124 @@ Select basic example
 
 https://user-images.githubusercontent.com/79102959/134949537-3647437e-0330-4692-bd30-ef6aa319bd7b.mov
 
+# Accessibility
+
+The accessibility requirements for all the form tags are already configured in the library. For components (input, checkbox, select, radio…) different attributes have been introduced that can be configured through props. 
+
+Next, we can see different attributes and tags to adjust the accessibility of the form and how to configure them: 
+
+| Attribute/Tag  	| Description  	
+|---	   |---	|
+|   Aria-describedby	  |  To configure that, only it is necessary to pass “name form” as a prop |
```

Comment:
```suggestion
|   Aria-describedby	  |  To configure that, only it is necessary to pass “name form” as a prop |
```
|   Aria-describedby	  |  To configure that, it's necessary to pass “name form” as a prop |